From: =?utf-8?q?S=C3=A9bastien_Villemot?= <sebastien@debian.org>
Date: Wed, 5 Feb 2020 00:00:00 +0000
Subject: Disable a test in timer.impure on all architectures

Upstream disabled it on i386, but it also fails randomly on amd64 and arm64.
(e.g. on arm64: https://ci.debian.net/data/autopkgtest/testing/arm64/s/sbcl/4152499/log.gz)

Bug: https://bugs.launchpad.net/sbcl/+bug/1221339
Bug-Debian: https://bugs.debian.org/919806
---
 tests/timer.impure.lisp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/timer.impure.lisp b/tests/timer.impure.lisp
index 25749d8..2b1c13c 100644
--- a/tests/timer.impure.lisp
+++ b/tests/timer.impure.lisp
@@ -272,7 +272,7 @@
 
 (with-test (:name (:timer :threaded-stress)
             :skipped-on (not :sb-thread)
-            :broken-on :x86
+            :broken-on :sbcl
             :fails-on :win32)
   #+win32
   (error "fixme")
